SwiftExercises.playground/section-1.swift

```diff
@@ -53,17 +66,16 @@ let picard = [
     "information": "Captain of the USS Enterprise",
     "favorite drink": "tea, Earl Grey, hot"]
 
-let characters = [worf, picard]
 
 func favoriteDrinksArrayForCharacters(characters:Array<Dictionary<String, String>>) -> Array<String> {
     // return an array of favorite drinks, like ["prune juice", "tea, Earl Grey, hot"]
     // WORK HERE
-    return []
+    var favDrinks = [worf["favorite drink"], picard["favorite drink"]]
```

Comment:
While this works, it is just as a hack as the solution you had in the dictionary assignment. Follow the same pattern from your final solution there: Create an empty mutable array, loop over all characters in the input array, grab the favorite drink and put it into the result array, return the result array in the end.